src/Yaml/NodeTraversal.ts

```diff
@@ -1,7 +1,7 @@
 import vscode from 'vscode';
 import { Node } from './Node';
 import { SubStack } from '../common/SubStack';
-import { cloneDeep } from 'lodash';
+import clone from 'lodash.clonedeep';
```

Comment:
Well, that was unexpected. Using the vscode developer tools I found that when vscode attempted to load the extension, an error about not finding the `lodash` module was thrown.

I have no idea how to test that, really...